### PR TITLE
Harmonize usage of --dry-run vs --dryrun in create-sibling-gitlab

### PR DESCRIPTION
**Pull request by adswa**
This adds a ``--dry-run`` parameter to ``create-sibling-gitlab`` to match the corresponding parameter in ``create-sibling-{github,gin,gogs,gitea}``, and deprecates the current ``--dryrun`` option.
The PR also includes a small doc fix to the URL pointing to the documentation of python gitlab

### CODE REVIEW

**Comment by yarikoptic** [comment]
+1 on harmonizing into `--dry-run`  (instead of `--dryrun`)

File: datalad/distributed/create_sibling_gitlab.py [hunk 1]
```diff
@@ -190,14 +191,17 @@ class CreateSiblingGitlab(Interface):
...
         dryrun=Parameter(
             args=("--dryrun",),
             action="store_true",
-            doc="""If this flag is set, no communication with GitLab is
-            performed, and no repositories will be created. Instead
-            would-be repository names and configurations are reported for all
-            relevant datasets
-            """),
+            doc="""Deprecated. Use the renamed
+        [CMD: --dry-run CMD][PY: `dry_run` PY] parameter""")
```
Comment (mih) [inline datalad/distributed/create_sibling_gitlab.py:204]: Is this a whitespace issue?

**Review by mih — CHANGES_REQUESTED** [review]
Makes sense! Thx!

It would be good to discontinue the use of `dryrun` in the tests, though.

**Comment by adswa** [comment]
> It would be good to discontinue the use of `dryrun` in the tests, though.

Right, thanks much for catching this. I have replaced them, and force-pushed.

File: datalad/distributed/create_sibling_gitlab.py (hunk 1 above)
Comment (adswa) [reply datalad/distributed/create_sibling_gitlab.py]: Thanks for catching this - a left over from copy-pasting. Should be resolved now.

**Comment by adswa** [comment]
The Appveyor failure is a network glitch, I think

> Error making request with Error Code NotFound and Http Status Code NotFound. No further error information was returned by the service.

**Review by mih — APPROVED** [review]
Thanks for the update!

**Pull request merged**